spec/javascripts/health-care/components/AddressState.spec.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactTestUtils from 'react-addons-test-utils';
+import AddressState from
+    '../../../../_health-care/_js/components/personal-information/AddressState';
+
+describe('<AddressState>', () => {
+  let component = null;
+
+  beforeEach(() => {
+    component = ReactTestUtils.renderIntoDocument(
+      <AddressState state={{ state: 'CA' }} onUserInput={(_unused) => {}}/>
```

Comment:
to be clear since I just realized `state` shows up twice in the code...what I'm expecting is
`<AddressState value="CA" />`